components/cas-inbound-authenticator/pom.xml

```diff
@@ -126,14 +126,9 @@
                             org.eclipse.equinox.http.helper,
                             org.joda.time;version="${joda.wso2.osgi.version.range}",
 
-                            org.opensaml.common,
-                            org.opensaml.saml2.core.*,
-                            org.opensaml.xml,
-                            org.opensaml.xml.util,
-                            org.opensaml.xml.validation,
+                            org.opensaml.saml.saml2.core.*,
```

Comment:
Let's not use * imports

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-cas/pull/48/commits/8f133856d762a1cda3c2367a29f15db7829827ce.